update readme (#22)
TEAM-VAAR/TEAM-VAAR: README.md

# README

> # honestreviews.com

Reviewer: Is this the deployed website?  If not, make sure to include it.

> honestreviews is an website for users to find honest up to date information on schools they might be interested in attending. We use ruby on rails and postgres for the backend and use React for the website front end. 

Reviewer: A few small recommended changes:
```suggestion
Honestreviews is a website for users to find honest, up-to-date information on schools they might be interested in attending. 
```
Keep in mind that while the front end is React, the application build is React in Rails.  The way this is worded appears more like this is a decoupled app which it is not.

Current features of the project include:
- creating a user account
- creating a review for one of the pre added schools
- update and delete a review you have created
- view reviews other users have left

Steps To Run Application: 

1. git clone application into local directory
2. cd into the application directory
3. to install react dependencies run $ yarn
4. to install ruby dependencies run $ bundle
5. to setup the postgres database run $ rails db:reset

# About TEAM VAAR 

> TEAM VAAR is a team of creative and technical professionals focused on providing a highly functional and polished product to their users. Find out more on our About Us page. 

Reviewer: I would recommend at least providing your names in here.
